### DISCUSSION TITLE
Modeling of aerodynamic interaction effects between rotor and fuselage

### DISCUSSION THREAD

**Discussion by ilm71** [byuflowlab/FLOWUnsteady#144] [General]
Hi all, 

following the discussion about importing openVSP-geometry like a whole plane in the tutorials, a question popped up regarding modelling the aerodynamics interaction effects between the rotor and the fuselage, e.g. of a VTOL. When I tried to implement a dummy fuselage from openVSP to FLOWUnsteady I figured out, that the downwash of the rotor still penetrates the "inner" volume of the VTOL fuselage ... is there any script available, which highlights how the imported grid can be converted by the usage of FLOWPanel to a non-lifting body easily? Or are there several steps as mentioned in the FLOWPanel prototype-branch necessary?
Thanks for the support!

**Comment by EdoAlvarezR** [comment]
Hi Phil!

As of now OpenVSP can only be used to generate degenerate geometry (like a VLM wing). The 3D geometry is transparent to the transparent to the solver, as you have noticed.

I have it in my todo list to also take the full 3D geometry and couple it with FLOWPanel, but it's uncertain when I would have the time to tackle that. If you think that would like to work on that, I'd be happy to send you the scripts that we use with the `flowpanel-prototype` branch, which you could use as a starting point.

**Reply by ilm71** [reply]
Hi Ed, 

thank you very much for your reply and also of your explanation. It would be great to have a look into your currently available scripts and look forward to your inputs! 

Best

**Reply by EdoAlvarezR** [reply]
Could you send me an email to Edo.AlvarezR@gmail.com and continue the conversation there?
